src/components/NotFound/NotFound.tsx

```diff
@@ -11,106 +12,98 @@ import Logo from '../../lib/Logo';
 import { PrimaryButton } from '../Button';
 
 type Props = {
-  className?: string,
-  onReturnHomeClick: () => void,
-  statusCode: number,
+  className?: string;
+  onReturnHomeClick: () => void;
+  statusCode: number;
 };
 
-class NotFound extends React.Component<Props> {
-  closeButton: Element | Text | null;
+const NotFound: React.FC<Props> = ({ className, onReturnHomeClick, statusCode }) => {
+  useEffect(() => focus(), []);
 
-  manageFocus = ({ nativeEvent }) => {
+  const manageFocus = ({ nativeEvent }) => {
```

Comment:
This function be wrapped with `React.useCallback(…)`, otherwise the function object will be re-generated on each rendering.